teachertool/src/components/MainPanel.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+// eslint-disable-next-line import/no-internal-modules
+import css from "./styling/MainPanel.module.css";
+
+import DebugInput from "./DebugInput";
+import MakeCodeFrame from "./MakecodeFrame";
+import EvalResultDisplay from "./EvalResultDisplay";
+import ActiveRubricDisplay from "./ActiveRubricDisplay";
+import SplitPane from "./SplitPane";
+
+interface IProps {}
+
+const MainPanel: React.FC<IProps> = () => {
+    return (
+        <div className={css["main-panel"]}>
+            <SplitPane split={"vertical"} defaultSize={"80%"} primary={"left"}>
+                {/* Left side */}
+                <>
```

Comment:
Similar question to as above. Why do we want empty tags in components? Would having something like this cause problems for screen readers?

Reply:
This is another instance of needing a logical container that effectively disappears at runtime. In this case, each fragment acts as an envelope to deliver content to each side of the split pane. After that, their job is done.